docs/owo/recipe-remainders.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Recipe Remainders
+project: owo
+---
+
+This feature allows for recipe specific Remainders within a specific Recipe. Minecraft has an internal Remainder system, but the downside to such is it is global for all recipes 
+
+Let's start with normal recipe you might make within your mod:
+
+```json title="test_recipe.json"
+{
+  "type": "minecraft:crafting_shaped",
+  "key": {
+    "#": {
+      "item": "minecraft:stick"
+    },
+    "X": {
+      "item": "minecraft:sand"
+    }
+  },
+  "pattern": [
+    "XXX",
+    "X#X",
+    "XXX"
+  ],
+  "result": {
+    "item": "minecraft:bedrock"
+  }
+}
+```
+
+To start, all remainders must be put within the `"owo:remainders"` JSON Object. Then such is as simple as declaring the identifier of the given target item first and then declaring the remainder item as either an identifier or as a Son Object with following Parameters:
```

Comment:
This needs to be a bit more concise, Son.
If you use a JSON Object over a direct Item Identifier, can you attach any NBT to the remainder?

Reply:
Besides it being a "Son Object", I give the following parameters that are accepted. I could see the possible confusion but I don't mention anything about NBT data.

Reply:
Besides it being a "Son Object", I give the following parameters that are accepted. I could see the possible confusion but I don't mention anything about NBT data.